Opera/OperaURLProvider.py

```diff
@@ -44,21 +45,16 @@ def get_opera_url(self, url):
         # Get list of links from directory listing
         page = self.download(url).decode('utf-8')
         links = re.findall(r'<a.*?\s*href="(.*?)".*?>', page)
-
-        # Evaluate the links in reverse alphabetical order
-        while True:
-            curl_cmd = self.prepare_curl_cmd()
-            curl_cmd.extend(["--head", url + max(links) + "mac/"])
-            output = self.download_with_curl(curl_cmd)
-            if "200 OK" in output:
-                version = max(links).rstrip("/")
-                break
-            links.remove(max(links))
-            if not links:
-                raise ProcessorError("Did not find any valid versions to download.")
+        latest = LooseVersion("0.0.0")
```

Comment:
You might consider using `APLooseVersion` instead, since `distutils.version.LooseVersion` is [deprecated](https://peps.python.org/pep-0632/).

Reply:
Thanks for the comment @homebysix! Where can I find docks on APLooseVersion? 🤔 I'm not sure where to import that from.

Reply:
Ah, I found it in autopkglib.

Reply:
Yep, that's the one. [Here's](https://github.com/autopkg/autopkg/blob/master/Code/autopkglib/GitHubReleasesInfoProvider.py#L21) a usage example.